pkg/api/types.go

```diff
@@ -59,6 +59,15 @@ type Config struct {
 	// BuilderBaseImageVersion provides optional version information about the builder base image.
 	BuilderBaseImageVersion string
 
+	// RuntimeImage specifies the image that will be a base for resulting image
+	// and will be used for running an application. By default, BuilderImage is
+	// used for building and running, but the latter may be overridden.
+	RuntimeImage string
+
+	// RuntimeArtifacts specifies a list source/destination folders that will
```

Comment:
just folders or files and folders?  can both the source and destination be files?